packaging/must-gather/must-gather

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Check if the script is run as root
+if [ "$EUID" -ne 0 ]; then
+  echo "This script must be run as root. Exiting."
+  exit 1
+fi
+
+echo "Warning: This operation could generate a very large file."
+read -p "Do you have enough storage space? (y/n): " -n 1 -r
+echo
+
+# Check the user's response
+if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+    echo "Operation canceled by the user."
+    exit 1
+fi
+
+timestamp=$(date +"%Y%m%d-%H%M%S")
+temp_dir=$(mktemp -d)
+
+journalctl --since "24 hours ago" > "$temp_dir/journal.log"
+
+cp /var/lib/flightctl/*.json "$temp_dir/"
+
+output_file="$temp_dir/system_info.txt"
+
+echo "Collecting system information..."
```

Comment:
Would you like to add progress before this?  "Gathering logs..."?  (Just because this is the first progress message and it's on the 3rd step)